AY2122S1-CS2103T-T11-2/tp
# Enhance UI

### Description

Closes #103.
- Added language field to the table display for all flashcards
- Added Slideshow component to display flashcards one at a time when a user is testing themselves
  - Currently contains placeholder values that will be removed once the `Test` command is implemented
  - Temporary button to toggle the slideshow component
  - Flashcards in the table are now not flipped due to the addition of the slideshow

### Type of change


### How to test


- Toggle the slideshow component by clicking on the button
- Add, edit, delete flashcards to ensure that language is accurately reflected in the GUI

### Checklist
